deb/build/debian/jenkins.init

```diff
@@ -129,7 +129,7 @@ do_start()
 # 
 get_running() 
 {
-    return `ps -U $JENKINS_USER --no-headers -f | egrep -e '(java|daemon)' | grep -c . `
+    return `ps -U $JENKINS_USER --no-headers -f | egrep -e '(java)' | grep -v defunct | grep -c . `
```

Comment:
why would we have defunct processes?  the launch should be detached and running as a deamon?